bcm/blockchain.go

```diff
@@ -29,10 +29,12 @@ type BlockchainInfo interface {
 	LastCommitDuration() time.Duration
 	LastBlockHash() []byte
 	AppHashAfterLastBlock() []byte
-	// Gets the BlockHash at a height (or nil if no BlockStore mounted or block could not be found)
+	// BlockHash gets the hash at a height (or nil if no BlockStore mounted or block could not be found)
 	BlockHash(height uint64) ([]byte, error)
```

Comment:
Remove this - it is unused now

Reply:
It is used here > https://github.com/hyperledger/burrow/blob/master/execution/evm/contract.go#L464

Reply:
It is still used here > https://github.com/hyperledger/burrow/blob/master/execution/evm/contract.go#L464